extensions/src/AWSSDK.Extensions.Configuration.AWSSystemsManager/AWSSDK.Extensions.Configuration.AWSSystemsManager.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>AWSSDK.Extensions.Configuration.AWSSystemsManager</AssemblyName>
+    <RootNamespace>Amazon.Extensions.Configuration.AWSSystemsManager</RootNamespace>
+    <ApplicationIcon />
+    <OutputType>Library</OutputType>
+    <StartupObject />
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.3.0" />
```

Comment:
the versions here are inconsistent with the version ranges in nuspec.

Reply:
Not sure what you mean. Both package references fall in the range defined
`<PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.3.0" />`
`<dependency id="AWSSDK.Extensions.NETCore.Setup" version="[3.3.0, 3.4)" />`
3.3.0 is >= 3.3.0 & < 3.4

`<PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="3.3.15" />`
`<dependency id="AWSSDK.SimpleSystemsManagement" version="[3.3.15, 3.4)" />`
3.3.15 is >= 3.3.15 & < 3.4

I'm I incorrect?

Reply:
The problem will go away once we remove nuspec and use csproj to drive nuget package generation.  However, I was confused why the nuspec defines a range but the csproj doesn't.  I thought you could express a range of versions as a dependency in csproj?

Reply:
I've checked and PackageReference doesn't currently support ranges, but does support wildcards
[package-references-in-project-files](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files)
[Open Issue for allowing ranges](https://github.com/NuGet/docs.microsoft.com-nuget/issues/869)